cmd/vpn_client/app/pathcontroller/ping.go

```diff
@@ -66,7 +67,7 @@ func (p *icmpPinger) pingWithTimer(client net.IP) error {
 			if errors.As(err, &neterr) && neterr.Timeout() {
 				err = fmt.Errorf("i/o timeout")
 			}
-			p.log.Info("ping failed", "ip", client, "duration", fmt.Sprintf("%dms", d.Milliseconds()), "error", err)
+			p.log.Info("ping failed", "ip", client, "duration", fmt.Sprintf("%dms", d.Milliseconds()), "error", err, "try", fmt.Sprintf("%d/%d", try, tries))
```

Comment:
Why don't we log the error at the call site, instead of adding a new argument to the function?